src/mongo/tree/MongoAccountTreeItem.ts

```diff
@@ -78,6 +79,20 @@ export class MongoAccountTreeItem implements IAzureParentTreeItem {
 
         throw new UserCancelledError();
     }
+
+}
+
+function validateDatabaseName(database: string): string | undefined | null {
```

Comment:
I don't see it used elsewhere. Specifying it as `private validateDatabaseName` would help.

Reply:
Prashanth, that's the module level function. Should I move it into class and then make private?

Reply:
Hmm. It seems private to the file since there's no export. This should be fine. :-)